angular.json

```diff
@@ -94,7 +95,13 @@
                             "node_modules/codemirror/lib/codemirror.css",
                             "node_modules/ol/ol.css"
                         ],
-                        "scripts": []
+                        "scripts": [],
+                        "vendorChunk": true,
+                        "extractLicenses": false,
+                        "buildOptimizer": false,
+                        "sourceMap": true,
+                        "optimization": false,
```

Comment:
why no optimization?

Reply:
As far as I see this is false for the default context (dev) but set to true in the **production** case.